src/main/java/com/brandwatch/internship/fetchandstoretweets/entities/Mention.java

```diff
@@ -20,7 +22,7 @@
     private String source;
     private long queryId;
 
-    public Mention(long id, String idStr, String text, Date createdAt, String fromUser, String profileImageUrl,
+    private Mention(long id, String idStr, String text, Date createdAt, String fromUser, String profileImageUrl,
```

Comment:
What's `idStr` for?

Reply:
I had the same question :-). It is a field in the Tweet class so I just kept it. It is strange that they have two identical fields for the id one is long and the other is string where they could just keep one and compute the other. I can remove it if you want. I do not think that it would cause any problems.